tutorials/relay.md

```diff
@@ -99,66 +96,47 @@ yes
 1. Delete the management VM, you no longer need it.
 1. Open SSH to the relay VM.
 1. Download the relay configuration files.
-+
 ```bash
 curl https://raw.githubusercontent.com/opengsn/gsn/master/dockers/relaydc/.env > .env
 mkdir config
 curl https://raw.githubusercontent.com/opengsn/gsn/master/dockers/relaydc/config-sample/gsn-relay-config.json > config/gsn-relay-config.json
 ```
 1. Edit `.env`:
-+
 ```bash
 nano .env
 ```
 1. In `.env`, specify:
-+
-|===
-| Parameter | Value
-
-| HOST
-| Your host name
-|===
+<table>
+<tr><th>Parameter</th><th>Value</th></tr>
+<tr><td>HOST</td><td>Your host name</td></tr>
+</table>
 1. Press Ctrl-O and then Enter to save the modified file.
 1. Press Ctrl-X to exit.
 1. Edit `config/gsn-relay-config.json` to specify:
-+
-|===
-| Parameter | Value
-
-| baseRelayFee
-| The base fee that your relay will charge
-
-| pctRelayFee
-| The percent fee that your relay will charge
-
-| versionRegistryAddress
-| The address for the version registry on the network you are using. 
-  [See this list](../deployments/networks.md).
-
-| ethereumNodeUrl
-| The URL to a node on the network you wish to use. If you do not know what to put here,
-  get a [free Infura account](https://infura.io), create a project, and look at 
-  **KEYS > ENDPOINTS** for your network. Use the endpoint that starts with https://
-|===
+<table>
```

Comment:
don't wrtite html.
put "markdown" table (yes, the old table was adoc, which is different)

Reply:
I tried a markdown table first. It didn't work with the formatting I wanted to put there.

Reply:
Markdown tables reset the ordered list I use for steps.

![image](https://user-images.githubusercontent.com/12722969/103158175-aedc6f00-4780-11eb-8da9-9fba5bee6997.png)

Reply:
Works property with Vuepress, so I can use md tables.